users/management/commands/send_user_auth_service_is_changing_notifications.py

```diff
@@ -0,0 +1,81 @@
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.dateparse import parse_datetime
+
+from users.services import AuthServiceNotificationService
+
+
+class Command(BaseCommand):
+    help = "Sends user authorization service is changing notifications."
+
+    def add_arguments(self, parser):
+        # user_joined_before: Optional[datetime] = None, obsoleted_users_only=True
```

Comment:
It is unclear IMHO what this comment is about. Is this the default behavior or something else?

Reply:
Accident. I'll remove it.